packages/mongo/mongo_connection.js

```diff
@@ -85,6 +85,10 @@ export const MongoConnection = function (url, options) {
     }
   }));
 
+  self.client.on('connectionPoolCleared', Meteor.bindEnvironment(event => {
+    console.log('connectionPoolCleared', event);
+  }));
```

Comment:
This log should be out, right?

Reply:
https://github.com/meteor/meteor/issues/13108#issuecomment-2491344054

To get community verification, I’d like it to appear so feedback can confirm if they see it. If they do, it suggests the process causing the crash is triggered and likely resolved.

This comment is a good reminder to remove before merging. But before it I would like other pair of eyes confirming it.